compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SemanticAnalyzer.java

```diff
@@ -666,6 +672,14 @@ public void visit(BLangSimpleVariable varNode) {
         BType lhsType = varNode.symbol.type;
         varNode.type = lhsType;
 
+        // Configurable variable type must be a subtype of anydata&readonly.
+        if (configurable && varNode.typeNode != null) {
+            if (!(types.isAssignable(lhsType, symTable.anydataType) &&
+                    types.isAssignable(lhsType, symTable.readonlyType))) {
+                dlog.error(varNode.typeNode.pos, DiagnosticCode.CONFIGURABLE_VARIABLE_MUST_BE_ANYDATA_AND_READONLY);
+                return;
```

Comment:
We need to remove this one as well. Otherwise init expr validation is not happening.

Reply:
Done thanx